planning/autoware_obstacle_cruise_planner/src/node.cpp

```diff
@@ -871,9 +1109,14 @@ std::optional<CruiseObstacle> ObstacleCruisePlannerNode::createCruiseObstacle(
   const std::vector<TrajectoryPoint> & traj_points, const std::vector<Polygon2d> & traj_polys,
   const Obstacle & obstacle, const double precise_lat_dist)
 {
-  const auto & object_id = obstacle.uuid.substr(0, 4);
   const auto & p = behavior_determination_param_;
 
+  if (use_pointcloud_) {
+    return std::nullopt;
+  }
+
+  const auto & object_id = obstacle.uuid.substr(0, 4);
```

Comment:
Please reduce diff if possible.
```suggestion
  if (use_pointcloud_) {
    return std::nullopt;
  }
  
  const auto & object_id = obstacle.uuid.substr(0, 4);
  const auto & p = behavior_determination_param_;
```

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/6907/commits/0c913b239d1ee9f54a2374c6e0b2c5a42fac78cf